### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by CharlesCZ**
Minor rewording

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ token,user,uid,"group1,group2,group3"
 
 When using bearer token authentication from an http client, the API
 server expects an `Authorization` header with a value of `Bearer
-THETOKEN`.  The bearer token must be a character sequence that can be
+token`.  The bearer token must be a character sequence that can be
```
Comment (reylejano) [inline content/en/docs/reference/access-authn-authz/authentication.md:124]: suggestion to show to replace the value with the actual token
```suggestion
<token>`.  The bearer token must be a character sequence that can be
```

Comment (CharlesCZ) [reply content/en/docs/reference/access-authn-authz/authentication.md]: Good idea, thanks :)

**Comment by tengqm** [comment]
@CharlesCZ Please squash your commits into one. Thanks.

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ token,user,uid,"group1,group2,group3"
 
 When using bearer token authentication from an http client, the API
 server expects an `Authorization` header with a value of `Bearer
-THETOKEN`.  The bearer token must be a character sequence that can be
+`<token>`.  The bearer token must be a character sequence that can be
```
Comment (reylejano) [inline content/en/docs/reference/access-authn-authz/authentication.md:124]: the first opening back tick is in the previous line
```suggestion
<token>`.  The bearer token must be a character sequence that can be
```

**Comment by CharlesCZ** [comment]
> @CharlesCZ Please squash your commits into one. Thanks.

squashed

**Pull request merged**